app/services/profiles/update.rb

```diff
@@ -4,8 +4,9 @@ class Update
     using HashAnyKey
     include ImageUploads
 
-    CORE_PROFILE_FIELDS = %i[summary brand_color1 brand_color2].freeze
```

Comment:
I moved these fields to a new constant because they are settings now, not profile fields.